Makefile

```diff
@@ -461,6 +461,10 @@ test-all: test-build ## Run everything in test/.
 test-all-valgrind: test-build
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=debug,release --valgrind
 
+.PHONY: test-all-suites
+test-all-suites: | bench-addons-build
```

Comment:
Maybe add a `##` comment so that this target appears in `make help`?

I kind of also expect `test-all` to run everything in `test/` given its help text. What isn't currently run by `test-all`?

Reply:
`test-all` only runs the default test suite so it does not run the suites listed in `IGNORED_SUITES` in `test.py`. Currently, that's these suites:

 'addons',
  'benchmark',
  'doctool',
  'internet',
  'js-native-api',
  'node-api',
  'pummel',
  'tick-processor',
  'v8-updates'tests'

`test-all` runs both Debug and Release, which is not intuitive to me. It assumes you have both builds, which many people won't. (I didn't.) And building and running Debug builds will be sloooooowwwwww. I don't want to change it though, because I don't know if it's integral to someone else's workflow.

Reply:
Regarding ^^^^^, I think the `-all` part in `test-all` means "test all types of builds, namely Debug and Release". That's not intuitive to me and seems like an edge case. It was added all the way back in 2009 in https://github.com/nodejs/node/commit/861f28f37a2. 

Maybe we can repurpose it after all? Given it's long history, it seems likely no one is actually using it.

Reply:
> Maybe add a ## comment so that this target appears in make help?

I added a comment and modified the one in `test-all` to more accurately reflect what it does.